index.js

```diff
@@ -526,21 +526,24 @@
 		 * Return focus to the trigger that opened the modal dialog.
 		 * If the trigger doesn't exist for some reason, move focus to
 		 * either the <main>, or <body> instead.
+		 * Note: Wait a tick before setting focus. See https://github.com/WICG/inert#performance-and-gotchas
 		 * Reset initialTrigger and activeModal since everything should be reset.
 		 */
-		if ( trigger !== null ) {
-			trigger.focus();
-		}
-		else {
-			if ( main && !returnToBody ) {
-				main.tabIndex = -1;
-				main.focus();
+		Promise.resolve().then(function() {
```

Comment:
so the only issue i have with this right now is that prior to this change, the script worked with IE11 out of the box.

can this be done in a way to avoid having to polyfill promise?  or would the script need a new dependency like [the promise polyfill?](https://www.npmjs.com/package/promise-polyfill)